frontend/src/lib/components/PropertyIcon.tsx

```diff
@@ -22,6 +22,14 @@ import { Tooltip } from 'lib/lemon-ui/Tooltip'
 import { HTMLAttributes, ReactNode } from 'react'
 import { countryCodeToFlag } from 'scenes/insights/views/WorldMap'
 
+const osIcons = {
+    ['Mac OS X']: <IconMacOS />,
```

Comment:
```suggestion
    ['macOS X']: <IconMacOS />,
```

Reply:
🤔 makes me think that we don't need this to be case sensitive... like there's not going to be mAcOsX that isn't macOS X

Reply:
See comment https://github.com/PostHog/posthog/pull/18973/files#r1410441083